sycl/doc/design/spirv-extensions/SPV_INTEL_function_variants.asciidoc

```diff
@@ -494,7 +494,7 @@ Add to Section 3.56.9, Function Instructions:
  +
  Make a copy of 'Operand X' if 'Condition X' is *true*.
 
- From all 'Condition X'-'Operand X' pairs, 'exactly' one 'Condition X' must be *true*. Consequently, at least one 'Condition X'-'Operand X' pair must be present.
+ From all 'Condition X'-'Operand X' pairs, at least one 'Condition X' must be *true*. Consequently, at least one 'Condition X'-'Operand X' pair must be present.
```

Comment:
In the uses cases I can think of, setting more than one _Condition X_ to true would indicate that something went wrong.  What is the rationale for allowing this and arbitrarily using the first condition?  Should we instead say this is not allowed and add a validation check?